### DISCUSSION TITLE
Mobile push conflict

### DISCUSSION THREAD

**Discussion by Vinzent03** [Vinzent03/obsidian-git#285] [Problems] [answered]
From https://github.com/denolehov/obsidian-git/issues/57#issuecomment-1225887955
> good news, the fix did work! unfortunately it fails to push due to
conflicts.  “Cannot push. You have conflict files.”  The current “test.md”
on github contains “testing” while the local file on my phone is as shown
in the screenshot before.  I’m not sure why this would cause issues tho as
the point of the plug-in is to be able to edit notes and sync?  Is there
something else I should have set up?

@dsr432
Did you change your remote repo after clone?

**Comment by dsr432** [comment]
no… i don’t even see an option to do that if i wanted to. It asked for the
remote in the initial setup but after that all i did was run the clone
command and change the backup intervals.

**Reply by Vinzent03** [reply]
Aaargh, should be fixed in 1.1.11

**Comment by dsr432** [comment, marked as answer]
Thank you so much! Everything is working perfectly now :)

**Reply by Vinzent03** [reply]
Great!
